### PR TITLE
Runbook Linting issues fixed

### PR DESCRIPTION
**Pull request by vrdmr**
Fixed the 200 response issue. Fixed the linting issues related to it. Also fixed the resource group binding.

----
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by sergey-shandar** [comment]
@ravbhatnagar new response code (200) has been added.

**Comment by vrdmr** [comment]
@ravbhatnagar Ping. Could you please review this PR. Thanks.

**Comment by ravbhatnagar** [comment]
Signing off from ARM. It should not be a breaking change since the 200 response is the final response. Service would be returning a 202 only. Also, there should not be any service side behavior change. Please Correct my understanding.

**Comment by vrdmr** [comment]
Yes. The service only returns 202 and there is no change done in the service. The 200 is only for fixing the Error statements during the linting.

**Comment by sergey-shandar** [comment]
@vrdmr please, resolve conflicts.

**Comment by vrdmr** [comment]
@sergey-shandar Resolved. Thanks.

**Review by sergey-shandar — CHANGES_REQUESTED** [review]

File: specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/definitions.json [hunk 1]
```diff
@@ -3753,7 +3753,7 @@
       "type": "string",
       "pattern": "^[-\\w\\._]+$",
       "description": "The resource group name.",
-      "x-ms-parameter-location": "method"
+      "x-ms-parameter-location": "client"
```
[the remark this answers is not in the document]
Comment (vrdmr) [reply specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/definitions.json]: We are planning to remove the ResourceGroupName and AutomationAccountName and put it only in the automationClient. We can set these values easily. Our SDK is still in preview and this is helping people to simplify the code.

**Review by sergey-shandar — APPROVED** [review]

**Pull request merged**